testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MsalWrapper.java

```diff
@@ -96,7 +96,7 @@ public void acquireTokenWithResource(@NonNull final Activity activity,
 
         final AcquireTokenParameters.Builder builder = getAcquireTokenParametersBuilder(activity, requestOptions, callback);
         builder.withAuthorizationQueryStringParameters(null);
-        builder.withResource(requestOptions.getScopes().toLowerCase().trim());
+        builder.withResource(requestOptions.getScopes().trim());
```

Comment:
Can we add a unit tests here that makes sure that the resource that we asked for here is present in the request that was generated?

Reply:
I haven't even run the existing unit tests yet, which this may have broken.  Nor have i testing with Broker... so lots of testing still to perform.